package-system/AwsIotDeviceSdkCpp/FindAwsIotDeviceSdkCpp.cmake.template

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project. For complete copyright and license terms please see the LICENSE at the root of this distribution.
```

Comment:
minor: a few files have copyright headers like this where it's all on one line rather than broken up into 2 lines

```
# Copyright (c) Contributors to the Open 3D Engine Project.
# For complete copyright and license terms please see the LICENSE at the root of this distribution.
```

Reply:
I only recently did a sweep of O3DE to correct this last friday. If you would like to use the script, i can give you the updated one that will correct this

Reply:
I'll just break the lines up.